### PR TITLE
add instructions about how to view/update secrets

### PR DESCRIPTION
**Pull request by afeld**
Yikes these steps are ugly 😝  Going to try and generalize other people's scripts for this, but wanted to write them down so that _I_ don't forget how.

/cc @18F/cloud-gov-ops

### CODE REVIEW

File: content/ops/secrets.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+Authorized federal staff rotate, encrypt, and backup keys monthly. Privileged users can access the keys only with two-factor authentication and a decryption passphrase. In the rare case that both the keys and the decryption passphrase for the backup are lost or compromised, new keys can be rotated in by authorized federal staff, while maintaining availability of information.
+
+### Grabbing AWS credentials
+
+Note that access here is gated by membership in [the cloud-gov-ci team](https://github.com/orgs/18F/teams/cloud-gov-ci) in GitHub.
+
+1. Go to https://ci.fr.cloud.gov/pipelines/terraform-provision (substitute relevant Concourse instance).
+1. Click the `bootstrap-tooling` job.
+1. Expand the `create-update-tooling` step.
+1. Grab `ci_access_key_id` and `ci_secret_access_key`.
```
Comment (afeld) [inline content/ops/secrets.md]: Would love to be able to get rid of this section --> https://github.com/18F/cg-atlas/issues/70

**Comment by afeld** [comment]
Updated!

**Comment by mogul** [comment]
> - /pipeline/build/public/ops/secrets/index.html
>   -  linking to internal hash #grabbing-aws-credentials that does not exist (line 497)
>     htmlproof 2.6.1 | Error:  HTML-Proofer found 1 failure!

**Comment by afeld** [comment]
Fixed!

**Pull request merged**